src/WordPress.php

```diff
@@ -164,6 +164,24 @@ public function __construct() {
 	 *
 	 * [--force]
 	 * : Resets the remote database if it is not empty.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Create WordPress site
+	 *     $ ee site create example.com --type=wp
+	 *
+	 *     # Create WordPress multisite subdir site
+	 *     $ ee site create example.com --type=wp --mu=subdir
+	 *
+	 *     # Create WordPress multisite subdom site
+	 *     $ ee site create example.com --type=wp --mu=subdom
+	 *
+	 *     # Create WordPress site with ssl from letsencrypt
+	 *     $ ee site create example.com --type=wp --ssl=le
+	 *
+	 *     # Create WordPress site with wildcard ssl
+	 *     $ ee site create example.com --type=wp --ssl=le --wildcard
+	 *
```

Comment:
Add example for remote database installation. It is a very important use case.

Reply:
Also add examples with `--admin-email`, `--admin-user`, `--admin-password` parameters, and other `wp-cli` flags that we are accepting for `wp core download` `wp config create` and `wp core install`

Reply:
Very important to have an example with `--locale` flag. It is a global flag and can be passed during WordPress site creation.